specs/data/text_file/countries.txt

```diff
@@ -0,0 +1,3 @@
+COUNTRY,ISO CODES
+Australia,61
+Bengaldesh,880
```

Comment:
It's not really an issue but Bengaldesh is not a country. Are you thinking of Bangladesh? Also these are the dialling codes not the ISO codes.